### DISCUSSION TITLE
macOS Terminal colors are completely off

### DISCUSSION THREAD

**Discussion by M8torr** [starship/starship#3236] [Q&A]
Hi Everyone,

I have just setup this MacBook a macOS Monterey clean install.
But starship is not displaying the correct colours in the default macOS Terminal compared to iTerm2.

The one in the background is in the Terminal.app. The one in the foreground is iTerm2.
![image](https://user-images.githubusercontent.com/1701084/141605428-2a17c4a7-94dc-42e0-957a-01abac867936.png)

Do you know if I am missing anything?

Thanks,
Marcos

**Comment by davidkna** [comment]
Are you using a Powerline/nerd font in Terminal.app?

**Reply by M8torr** [reply]
Yeah, it is FiraCode Nerd Font.

**Reply by gburlingame** [reply]
Hello - did you ever figure this out?  I'm having the same issue.

**Reply by M8torr** [reply]
No, I stick with the default theme after this.
I would love to have the other one.

**Reply by davidkna** [reply]
Terminal.app does not support 24-bit RGB colors (hex-colors in starship). It should work if you replace them with one of the named colors or 0-255 ANSI codes ([relevant docs](https://starship.rs/advanced-config/#style-strings)).

**Reply by valentin-pouget-forge** [reply]
https://github.com/kdeldycke/dotfiles/blob/main/starship-ansi-colors.py
